test/util/rpcauth-test.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+# Copyright (c) 2015-2018 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test share/rpcauth/rpcauth.py
+"""
+import base64
+import configparser
+import hmac
+import importlib
+import os
+import sys
+import unittest
+
+class TestRPCAuth(unittest.TestCase):
+    def setUp(self):
+        config = configparser.ConfigParser()
+        file_path = os.path.abspath(os.path.dirname(__file__)) + "/../config.ini"
```

Comment:
nit: use `os.path.join()` for maximum portability (instead of concat'ing path strings)

Reply:
Also, should probably say `config_path`

Reply:
Done